jbpm-human-task/jbpm-human-task-jpa/src/main/resources/META-INF/Taskorm.xml

```diff
@@ -1379,5 +1379,36 @@
         </query>
         <!-- hint name="org.hibernate.timeout" value="200"/ -->
     </named-query>
-    
+   
+    <named-query name="TasksByPotentialOwnerGroups">
+        <query>
+            select distinct 
+            new org.jbpm.services.task.query.TaskSummaryImpl(
+                t.id,
+                t.name,
+                t.description,
+                t.taskData.status,
+                t.priority,
+                t.taskData.actualOwner.id,
+                t.taskData.createdBy.id,
+                t.taskData.createdOn,
+                t.taskData.activationTime,
+                t.taskData.expirationTime,
+                t.taskData.processId,
+                t.taskData.processInstanceId,
+                t.taskData.parentId,
+                t.taskData.deploymentId,
+                t.taskData.skipable              )
+            from
+            TaskImpl t
+            join t.peopleAssignments.potentialOwners potOwners
+            where
+            t.archived = 0 and
+            potOwners.id in (:groupIds) and
+            TYPE(potOwners) = 'Group'
+            order by t.id DESC
```

Comment:
Minor thing, but the lines in the `from` and `where` blocks should be indented for better readability.